packages/tasit-act/src/tasitAct.test.js

```diff
@@ -0,0 +1,83 @@
+import Contract from "./tasitAct";
+import { expect, assert } from "chai";
+// Note: This file is originally genarated by `tasit-contracts` and was pasted here manually
+// See https://github.com/tasitlabs/TasitSDK/issues/45
+import { abi as contractABI } from "./helpers/SimpleStorage.json";
+// Note: Under the current `tasit-contracts` setup SimpleStorage aways will deployed with this address
+// See https://github.com/tasitlabs/TasitSDK/pull/59#discussion_r242258739
+const contractAddress = "0x6C4A015797DDDd87866451914eCe1e8b19261931";
+
+describe.skip("Contract", function() {
+  let simpleStorage;
+
+  beforeEach("should connect to and existing contract", async () => {
+    // Connection with blockchain
+    // Ethers.js allows sets a "connector" (provider or already connected wallet) on constructor of Contract
+    // or later with contract.connect() method.
```

Comment:
I think tasit-act should internally keep track of whether it's connected and establish a connection whenever it's needed, but it shouldn't be connected by default. I don't think `tasit-act` needs a `connect()` function

Reply:
I should add this ^ to the README for `tasit-act`

Reply:
Hm.., so devs will use our Infura API KEY?

Reply:
No, I don't think so. Maybe we expect that they put theirs in a `.env` file and we look for that by default

I think the behavior above can still work with their own Infura API key

Reply:
I made an issue for adding this to the README https://github.com/tasitlabs/TasitSDK/issues/72

Reply:
Yes, it works. I've investigated it for a while, but I haven't figured out how it works without API_KEY.

Reply:
I don't think it needs to work without an API_KEY. We'll just make the key configurable when devs are using `tasit-act`. Does that make sense? Ideally it will be configurable through a config file and not a method we expose.

Reply:
Yes, I agree.

Reply:
Related to: https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244828773

Reply:
Does this require a response, or are you just noting that for future reference?

Reply:
For future reference only.

Reply:
That's config is already supported and we have this issue either: https://github.com/tasitlabs/TasitSDK/issues/92